libs/designer/src/lib/ui/panel/connectionsPanel/createConnection/formInputs/connectionMultiAuth.tsx

```diff
@@ -11,12 +12,17 @@ export interface ConnectionMultiAuthInputProps {
 }
 
 const ConnectionMultiAuthInput = ({ isLoading, value, onChange, connectionParameterSets }: ConnectionMultiAuthInputProps) => {
+  const authType = useIntl().formatMessage({
```

Comment:
```suggestion
  const intl = useIntl()
  const authType = intl.formatMessage({
```

Format-js linter is very dumb, in fact it's letting you use an ID that isn't correct at all. It always needs to be intl.formatMessage and the ID needs to be a hash of the input(the linter auto fix takes care of this)